src/switcher-ui.tsx

```
@@ -1324,6 +1324,21 @@ function SwitcherApp() {
                             </span>
                           );
                         })()}
+                        {session.accountLabel && !session.accountIsDefault && (
```

ESLint

[error] Replace `{session.accountLabel·&&` with `··{session.accountLabel·&&⏎···························`

(prettier/prettier)